.devcontainer/init

```
@@ -1 +1,3 @@
+#!/bin/sh
+
```

Good catch. Think we also need a `set -e` here?

---

Nvmd, probably redundant -- spun it up and looks like Codespaces will swallow errors and launch a recovery VM if it finds any.